.travis.yml

```diff
@@ -8,8 +8,10 @@ dist: trusty
 language: generic
 compiler:
   - gcc
-#notifications:
-#  Whoever receives success and/or failure via email
+notifications:
+  email:
+    recipients:
+      - gm130s@gmail.com
```

Comment:
You have provided your mail address here, so adding a hint to change it for users that copied this config might prevent you from spam ;)